django_rq/views.py

```diff
@@ -51,6 +52,11 @@ def stats(request):
             queue_data['finished_jobs'] = len(finished_job_registry)
             queue_data['started_jobs'] = len(started_job_registry)
             queue_data['deferred_jobs'] = len(deferred_job_registry)
+            try:
+                scheduler = get_scheduler(queue.name)
+                queue_data['scheduled_jobs'] = len(scheduler.get_jobs())
```

Comment:
This line could potentially be very inefficient as it fetches all scheduled jobs. I'd rather we implement a `scheduler.count()` first before we merge this in.